django-data/image/validation/tests/test_tasks.py

```diff
@@ -91,18 +101,45 @@ def setUp(self):
 
         # another patch
         self.send_msg_ws_patcher = patch(
-                'validation.tasks.send_message_to_websocket')
+            'validation.tasks.send_message_to_websocket')
         self.send_msg_ws = self.send_msg_ws_patcher.start()
 
-    def check_async_called(self, message, notification_message, pk=1):
+        # mocking validation summary
+        self.create_vsummary_patcher = patch(
+            'validation.tasks.ValidateTask.create_validation_summary')
+        self.create_vsummary = self.create_vsummary_patcher.start()
+
+    def check_async_called(
+            self, message, notification_message, validation_message=None,
+            pk=1):
+
         """Check django channels async messages called"""
 
+        # defining default validation message.
+        if not validation_message:
+            animal_qs = Animal.objects.filter(
+                name__submission=self.submission)
+            sample_qs = Sample.objects.filter(
+                name__submission=self.submission)
+
+            validation_message = {
```

Comment:
Here I defined a default validation message, so it simpler to test in some methods

Reply:
👍